pkg/scheduler/partition_manager.go

```diff
@@ -120,6 +121,12 @@ func (manager *partitionManager) cleanQueues(queue *objects.Queue) {
 				log.Logger().Debug("unexpected failure removing the queue",
 					zap.String("partitionName", manager.pc.Name),
 					zap.String("queue", queue.QueuePath))
+			} else {
+				// we should clean the queue tracker for ths deleted queue
+				log.Logger().Debug("removing queueTracker",
+					zap.String("queueName", queue.QueuePath),
+					zap.String("partitionName", manager.pc.Name))
+				ugm.GetUserManager().CleanDeletedQueueTrackers(queue.QueuePath)
```

Comment:
Minor simplification:
```
		if queue.IsEmpty() {
			// all OK update the queue hierarchy and partition
			if !queue.RemoveQueue() {
				log.Logger().Debug("unexpected failure removing the queue",
					zap.String("partitionName", manager.pc.Name),
					zap.String("queue", queue.QueuePath))
                                return
			}
			log.Logger().Debug("removing queueTracker",
				zap.String("queueName", queue.QueuePath),
				zap.String("partitionName", manager.pc.Name))
                        ugm.GetUserManager().CleanDeletedQueueTrackers(queue.QueuePath)
		} ...
```

Reply:
Good suggestion, addressed in latest PR.